c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1150,14 +1150,14 @@ static std::string reportProcessMemoryInfo(int device) {
           pci_id, &nvml_device));
 
   std::vector<nvmlProcessInfo_v1_t> procs(8);
-  unsigned int size = procs.size();
+  std::vector<decltype(nvmlProcessInfo_v1_t)>::size_type size = procs.size();
```

Comment:
Couldn't this just be auto?

Reply:
Yes that would work also. I prefer explicit over implicit but sure I'll commit this.